commit 5b8758f42b0abeab051f6c71f2018e6b8bfdc0c3 (from 0183e6332a11547641d2029a7df435ab83db924b)
Merge: 0183e63 520f1a5
Author: ErikZalm <erik@vdzalm.eu>
Date:   Fri Mar 23 11:54:37 2012 -0700

    Merge pull request #126 from tommycheng/patch-1
    
    Fix compilation error when PIDTEMP is undefined and ULTIPANEL is defined...

diff --git a/Marlin/ultralcd.pde b/Marlin/ultralcd.pde
index c1214b5..e388890 100644
--- a/Marlin/ultralcd.pde
+++ b/Marlin/ultralcd.pde
@@ -1181,6 +1181,7 @@ void MainMenu::showControlTemp()
         }
         
       }break;
+      	#ifdef PIDTEMP
       case ItemCT_PID_P: 
       {
       if(force_lcd_update)
@@ -1323,6 +1324,7 @@ void MainMenu::showControlTemp()
         }
         
       }
+      	#endif
       #endif
       break;
     default:   

